scripts/check_copyright.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+header_length=$(wc -l < LICENSE_FILE_HEADER)
+cur_year=$(date +%Y)
+
+license_file="LICENSE_FILE_HEADER"
```

Comment:
Any particular reason to have that file rather than just have the header as a configurable string in that script?

Reply:
Good point. Changed.